src/index.js

```diff
@@ -145,6 +143,24 @@ export default function auth(config = {}) {
     // Register error handling middleware for redirecting to support
     // redirecting on authentication failure.
     app.use(middleware.failedLogin(authOptions));
+
+    // Setup route handler for default success redirect
+    if (authOptions.shouldSetupSuccessRoute) {
```

Comment:
What was the reason again for adding those? I mean if you do a redirect it will almost certainly be to your own success page no?